CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
```

Comment:
Keeping a changelog would indeed be nice. How about a more common format like shown e.g. [here](https://keepachangelog.com/en/1.1.0/) (which I also use for Flare).

Reply:
You forgot to link the `[Unreleased]`-section to the GitHub commits. But nothing really important.